src/python/ensembl/io/genomio/gff3/simplifier.py

```diff
@@ -207,18 +207,28 @@ def create_gene_for_lone_transcript(self, feat: SeqFeature) -> SeqFeature:
             return feat
 
         new_type = "ncRNA_gene"
-        if feat.type in ("tRNA", "rRNA"):
+        if feat.type in ("tRNA", "rRNA", "mRNA"):
             new_type = "gene"
         logging.debug(f"Put the transcript {feat.type} in a {new_type} parent feature")
         new_gene = SeqFeature(feat.location, type=new_type)
         new_gene.qualifiers["source"] = feat.qualifiers["source"]
         new_gene.sub_features = [feat]
 
         # Use the transcript ID for the gene, and generate a sub ID for the transcript
-        new_gene.id = feat.id
+        new_gene.id = f"{feat.id}_gene"
         new_gene.qualifiers["ID"] = new_gene.id
-        feat.id = self.stable_ids.generate_transcript_id(new_gene.id, 1)
-        feat.qualifiers["ID"] = feat.id
+
+        # Check if it's a pseudogene
+        if feat.type == "mRNA":
+            is_pseudo = False
+            for subfeat in feat.sub_features:
+                pseudo_qual = subfeat.qualifiers.get("pseudo", [""])[0]
+                if subfeat.type == "CDS" and pseudo_qual == "true":
+                    is_pseudo = True
+                    del subfeat.qualifiers["pseudo"]
+                    break
+            if is_pseudo:
+                new_gene.type = "pseudogene"
```

Comment:
What about the two lines that have been removed for `tRNA` and `rRNA`?
```
        feat.id = self.stable_ids.generate_transcript_id(new_gene.id, 1)
        feat.qualifiers["ID"] = feat.id
```

Reply:
Should I keep transfering the transcript ID to the gene for tRNAs and rRNA, but not mRNAs? Or transfer it for all?